.github/workflows/commit.yml

```diff
@@ -56,20 +56,17 @@ jobs:
     name: create services (development)
     environment: development
     runs-on: ubuntu-latest
-    container:
-      image: cloudfoundry/cli:8
-      env:
-        org: gsa-datagov
-        space: development
-        user: ${{secrets.CF_SERVICE_USER}}
-        password: ${{secrets.CF_SERVICE_AUTH}}
     steps:
       - name: checkout
         uses: actions/checkout@v2
-      - name: auth
-        run: ./cfauth.sh
```

Comment:
I missed this the first time.  Do we want to delete this file from the repo since it's not being used anymore?